pkg/runner/client.go

```diff
@@ -155,9 +156,25 @@ func CreateClient(cfg *Config, promMetrics *api.Metrics) (*pgclient.Client, erro
 	if !cfg.UseVersionLease {
 		leasingFunction = nil
 	}
+
+	multiTenancy := tenancy.NewNoopAuthorizer()
+	if cfg.TenancyCfg.EnableMultiTenancy {
+		var multiTenancyConfig tenancy.AuthConfig
+		if cfg.TenancyCfg.SkipTenantValidation {
+			multiTenancyConfig = tenancy.NewAllowAllTenantsConfig(cfg.TenancyCfg.AllowNonMTWrites)
+		} else {
+			multiTenancyConfig = tenancy.NewSelectiveTenancyConfig(cfg.TenancyCfg.ValidTenantsList, cfg.TenancyCfg.AllowNonMTWrites)
+		}
```

Comment:
```suggestion
		multiTenancyConfig := tenancy.NewAllowAllTenantsConfig(cfg.TenancyCfg.AllowNonMTWrites)
		if !cfg.TenancyCfg.SkipTenantValidation {
			multiTenancyConfig = tenancy.NewSelectiveTenancyConfig(cfg.TenancyCfg.ValidTenantsList, cfg.TenancyCfg.AllowNonMTWrites)
		}
```
Might even want to reverse the options, so we don't use negation in the if clause. Should be cleaner to read.

Reply:
I think this is better than the reverse. Selective config is a case that is non-default so.